saleor/checkout/templates/checkout/snippets/voucher-form.html

```diff
@@ -0,0 +1,33 @@
+{% load bootstrap_field bootstrap_button bootstrap_icon from bootstrap3 %}
+{% load trans from i18n %}
+{% load gross net tax from prices_i18n %}
+
+{% if form %}
+    <tr>
+        {% if discount %}
+            <td class="noborder">
+                <form action="{% url "checkout:remove-voucher" %}?next={{ request.path }}"
+                      method="post"
+                      class="form-inline">
+                    {% csrf_token %}
+                    <span class="text-success">{{ discount.name }}</span>
+                    <button type="submit"
+                            class="btn btn-link btn-xs"
+                            aria-label="{% trans "Remove" %}">
+                            {% bootstrap_icon "remove" %}
+                    </button>
+                </form>
+            </td>
+            <td class="noborder text-right">
+                -&nbsp;{% net discount.amount %}
```

Comment:
We can't just stick a minus in front of the string, we need to pass a negative value to formatter as where minus goes varies between locales.